get.sh

```diff
@@ -234,7 +234,7 @@ getBinaryOpenjdk()
 			elif [[ "$jar_dir_name"  =~ native-test-libs*  &&  "$jar_dir_name" != "native-test-libs" ]]; then
 				mv $jar_dir_name native-test-libs
 			#The following only needed if openj9 has a different image name convention
-			elif [[ "$jar_dir_name" != "j2sdk-image"  &&  "$jar_dir_name" != "native-test-libs" ]]; then
+			elif [[ "$jar_dir_name" != "j2sdk-image"  &&  "$jar_dir_name" != "native-test-libs" && "$jar_dir_name" != "jtreg" ]]; then
```

Comment:
I am not a fan of this temporary approach (to change get.sh and upload via overloading CUSTOMIZED_SDK_URL), when I think the real/right solution is so close.  I was assuming you would not put in a PR but wait to get the real jtreg for zos build hosted in the open.  

Is there a reason to push this temporary work in now versus waiting the few days it will take for the final solution?

Reply:
I don't think an open jtreg for zOS solution is days away, I suspect more like weeks....

Reply:
Is setting up a parameter for `jtreg_url` a bad idea, even if it's temporary and can be removed after said weeks? I know we don't really want more params. Is the end goal to host it at Adopt and have a simple switch between z/os and everything else?